src/services/dumpers/templates/agent-nodejs/Dockerfile.hbs

```
@@ -0,0 +1,16 @@
+FROM node:16
```

```suggestion
FROM node:18
```
As most of our stack is supposed to be 18 + it's the LTS

---

👍